hardware/plugins/Plugins.cpp

```diff
@@ -657,19 +658,64 @@ namespace Plugins {
 		if (m_bIsStarted) StopHardware();
 
 		//	Add start command to message queue
+		m_bIsStarting = true;
```

Comment:
I have to review the whole code, but why would you start something in a destructor ?

> A destructor is a special member function that is called when the lifetime of an object ends. The purpose of the destructor is to free the resources that the object may have acquired during its lifetime

Reply:
It's not in the destructor, it's in CPlugin::StartHardware()